### PR TITLE
'[REF] addons-vauxoo: Standardize project with odoo guidelines.'

### PR DESCRIPTION
**Pull request by KarenKawaii**

### CODE REVIEW

File: mrp_button_box/__openerp__.py [hunk 1]
```diff
@@ -24,10 +24,10 @@
 {
     "name": "MRP Button Box",
     "version": "1.6",
-    "author": "Vauxoo C.A.",
+    "author": "Vauxoo C.A.",  # pylint: disable=C8101
```
Comment (moylop260) [inline mrp_button_box/__openerp__.py]: Vauxoo*\* cambialo a `Vauxoo`, please
Haz un rebase, esta marcando conflictos

**Comment by KarenKawaii** [comment]
@moylop260  I did the rebase. But the branch has conflicts yet.

**Comment by moylop260** [comment]
you will need a `git rebase -i --autosquash vauxoo/8.0`
Where vauxoo is your vauxoo/addons-vauxoo remote.

**Comment by KarenKawaii** [comment]
Is it possible do a git rebase again?

**Comment by KarenKawaii** [comment]
@moylop260  If I understood well, the command should be like:
`git rebase -i --autosquash vauxoo/8.0-standardize-addons-vauxoo-pr1-dev-moylop260/8.0`
Or I'm wrong?

**Pull request merged**